istio-control/istio-config/templates/deployment.yaml

```diff
@@ -181,9 +181,28 @@ spec:
             readOnly: true
           - name: envoy-config
             mountPath: /var/lib/istio/galley/envoy
+          {{- if .Values.global.sds.enabled }}
+          - name: sds-uds-path
+            mountPath: /var/run/sds
+            readOnly: true
+          - name: istio-token
+            mountPath: /var/run/secrets/tokens
+          {{- end }}
 {{- end }}
 
       volumes:
+      {{- if .Values.global.sds.enabled }}
+      - hostPath:
```

Comment:
why a hostpath?  There is no good reason to use a host path in anywhere of the manifests of Istio with the possible exception of CNI.

Reply:
Yes we agree. The reason for doing that is in we are still using the node agent DaemonSet for SDS. We are working on the per-pod Istio Agent to replace the node agent, which will get rid of the hostPath.